### ISSUE TITLE
[Minor feature request] Easier way to check for supported controls/features

### ISSUE THREAD

**Issue by grawity** [cannatag/ldap3#594]
Right now `server.info.supported_{controls,features,etc.}` is an array of "descriptive" tuples:

```
py3.7 >>> pprint(conn.server.info.supported_features)
[('1.3.6.1.1.14', 'FEATURE', 'Modify-Increment', 'RFC4525'),
 ('1.3.6.1.4.1.4203.1.5.1', 'FEATURE', 'All Op Attrs', 'RFC3673'),
 <...>
 ('1.3.6.1.4.1.4203.1.5.5', 'FEATURE', 'language Range Options', 'RFC3866')]
```

…which means I cannot use simple `if (oid in features):` because I have to specify the whole tuple (whose contents might not be set in stone), so I loop over the list every time I need to check something:

```
ctrl_oids = {item[0] for item in conn.server.info.supported_controls}
feat_oids = {item[0] for item in conn.server.info.supported_features}
if "1.3.6.1.1.13.2" in ctrl_oids and "1.3.6.1.1.14" in feat_oids:
    ...
```

It would be more convenient if this were a dict keyed by the OID, because then it would become a trivial dict key lookup. But I assume it's way too late to change that now due to API compatibility. So it would be nice if _at least_ a helper method was present for all those `supported_*` sections, e.g.:

```
if conn.server.has_control("1.3.6.1.1.13.2") and conn.server.has_feature("1.3.6.1.1.14"):
    ...
```

Or if there already is one... is it documented?

**Comment by cannatag** [comment]
This looks like a very good idea. Will work on this in the next release of ldap3.

**Comment by cannatag** [comment]
Hi, I've added the has_control(), has_feature() and has_extension() methods in the server object. You can use it to query for the presence of a supported control/extension/feature either by ID or by the name in the relevant RFC. Can you check the code in dev?

**Comment by grawity** [comment]
Seems to work well, thanks.

**Comment by cannatag** [comment]
Fixed in version 2.6.
